tmk_core/common/chibios/eeprom_teensy.c

```diff
@@ -44,8 +44,8 @@
 
 #define F_CPU KINETIS_SYSCLK_FREQUENCY
 
-static int kinetis_hsrun_disable(void) {
 #if defined(MK66F18)
+static int kinetis_hsrun_disable(void) {
```

Comment:
Maybe do this instead:
```c
#if defined(MK66F18)
static int kinetis_hsrun_disable(void) {
   // ... the real implementation
}
#else
static inline int kinetis_hsrun_disable(void) { return 0; }
#endif
```
(`static inline` does not cause an error if the function ends up unused)

Reply:
> (static inline does not cause an error if the function ends up unused)

Fair point. Probably easier that way.

Reply:
Swapped the definitions to `static inline`, still broken with `FTFE_FCCOB0_CCOBn_SET`.